Makefile

```diff
@@ -62,6 +62,10 @@ build-date:
 	cd date_pkg && gox -osarch="linux/amd64 darwin/amd64 windows/amd64" -ldflags="-s -w" -output "main_{{.OS}}_{{.Arch}}"
 publish-date: build-date |
 	apptree publish package -d date_pkg --host ${HOST}
+build-mailgun: |
+			cd database/mailgun_pkg && gox -osarch="linux/amd64 darwin/amd64 windows/amd64" -ldflags="-s -w" -output "main_{{.OS}}_{{.Arch}}"
+publish-mailgun: build-mailgun |
+	apptree publish package -d database/mailgun_pkg --host ${HOST}
```

Comment:
I think the path is wrong. It looks like this is referencing `database/mailgun_pkg` but it looks like it should just be `mailgun_pkg`

Reply:
Good catch, updated